src/theme/styled.d.ts

```diff
@@ -44,6 +44,9 @@ export interface Colors {
   yellow1: Color
   yellow2: Color
   blue1: Color
+  purple: Color
+  border: Color
+  disabled: Color
```

Comment:
As this is just a type we should be able to extend it fairly easily anywhere in `/custom`
[Merging Interfaces](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces)